src/widgets/helper/SearchPopup.cpp

```diff
@@ -106,6 +106,14 @@ void SearchPopup::addChannel(ChannelView &channel)
     this->updateWindowTitle();
 }
 
+void SearchPopup::goToMessage(const MessagePtr &message)
+{
+    for (const auto &channel : this->searchChannels_)
+    {
+        channel.get().scrollToMessage(message);
```

Comment:
Early out

Reply:
Done in 4ed3937dc6e3351be95858a887720418147b5a32.